src/rust/Dockerfile

```diff
@@ -152,8 +152,9 @@ EXIT_STATUS=0
 
 for test in $(find /tests -type f -executable -exec readlink -f {} \;)
 do
-    echo Executing "${test}"
-    "${test}"
+    echo "--- Executing ${test}"
+    # Redirect stderr so it's inline with stdout
+    "${test}" 2>&1
```

Comment:
Why do you want to do this? One of the nice things about Nomad is that it splits stderr and stdout into separate panes so its relatively easy to look for errors.

Reply:
Gonna do this
```

if [[ ${EXIT_STATUS} -ne 0]]; then
  echo -e "\n❯❯ The following tests failed: ${FAILING_TESTS[@]}"
  echo -e "   See .stderr.txt (or the Nomad stderr tab) to see failure backtraces."
fi
```